before_deploy.sh

```diff
@@ -0,0 +1,10 @@
+# CHART_VERSION is computed from the TAG details of the commit. Every Github release creates tag with the release name.
```

Comment:
The first line here should probably be #!/bin/sh or #!/bin/bash to indicate the appropriate execution shell and reduce ambiguity.

Reply:
Fixed